pkg/cache/cache_test.go

```diff
@@ -0,0 +1,55 @@
+package cache_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/yorkie-team/yorkie/pkg/cache"
+)
+
+func TestCache(t *testing.T) {
+	t.Run("create lru expire cache test", func(t *testing.T) {
+		lruCache, err := cache.NewLRUExpireCache(1)
+		assert.NoError(t, err)
+		assert.NotNil(t, lruCache)
+
+		lruCache, err = cache.NewLRUExpireCache(0)
+		assert.Error(t, err)
+		assert.Nil(t, lruCache)
+	})
+
+	t.Run("add test", func(t *testing.T) {
+		lruCache, err := cache.NewLRUExpireCache(1)
+		assert.NoError(t, err)
+
+		lruCache.Add("request1", "response1", time.Second*5)
```

Comment:
Wouldn't it be more readable to write the unit after the value?
```suggestion
		lruCache.Add("request1", "response1", 5*time.Second)
```